### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1425,45 +1425,47 @@ ________
 
 Legal and Support
 ```````````````````````````
+Legal and Supprt links will be hidden in the user interface if these fields are left blank.
```
Comment (lindy65) [inline source/administration/config-settings.rst]: `Supprt` > `Support` ?

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -1222,9 +1222,9 @@ For deployments seeking additional tracking of system behavior using Segment.com
...
 
-Using a Google Developer Key allows Mattermost to detect when a video is no longer available and display the post with a *Video not found* label.
+Using a Google API Key allows Mattermost to detect when a video is no longer available and display the post with a *Video not found* label.
 
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | This features's ``config.json`` setting is ``"GoogleDeveloperKey": ""`` with string input.                                                                            |
```
Comment (lindy65) [inline source/administration/config-settings.rst]: Replace throughout document:
`features's' > `feature's`

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -1482,23 +1482,23 @@ Native App Links
...
+Configurable link to a download page for Native Apps. When a link is present, an option to "Download Mattermost Apps" will be added in the Main Menu so users can find the download page. Leave this field blank to hide the option from the Main Menu. Defaults to a page on about.mattermost.com where users can download the iOS, Android, and Desktop clients. If you are using an `Enterprise App Store <https://docs.mattermost.com/deployment/push.html?highlight=enterprise%20app#push-notifications-and-mobile-devices>`_ for your mobile apps, change this link to point to a customized download page where users can find the correct apps.
 
 +-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | This features's ``config.json`` setting is ``"AppDownloadLink": "https://about.mattermost.com/downloads/"`` with string input.                                                            |
```
Comment (lindy65) [inline source/administration/config-settings.rst]: Amend throughout doc:
`features's` should be `feature's`

**Comment by jasonblais** [comment]
Still needs Google/Office365 SSO config settings

**Comment by esethna** [comment]
Reviewed, +1, fixed some typoes and strange formatting of the google and office365 parts

**Comment by esethna** [comment]
Review complete, Jason is helping fix the conflicts

**Pull request merged**